estate-services/src/main/java/org/egov/ps/repository/PropertyQueryBuilder.java

```diff
@@ -106,6 +123,8 @@
 	public static final String RELATION_OWNER_DOCUMENTS = "ownerdocs";
 	public static final String RELATION_COURT = "court";
 	public static final String RELATION_BIDDER = "bidder";
+	public static final String RELATION_ESTATE_DEMAND = "estatedemand";
```

Comment:
Rename bothe of these to `finance` and use only one relation.

Reply:
Okay